src/App.svelte

```diff
@@ -0,0 +1,115 @@
+<script>
+	let display = 0;
+	let firstOperand = null;
+	let secondOperand = false;
+	let operator = null;
+	let operatorClicked = false;
+
+	const addDisplay = (num) => {
+		if(display === 0 || operatorClicked ) {
+			display = '';	
+		}
+		if(display.length <= 13){
+			display += String(num);	
+		}
+		operatorClicked = false;
+	}
+
+	const dot = () => {
+		if(display == 0) {
+			display = '0.';
+		}else if(!display.includes('.')){
```

Comment:
There should be spaces after `}` and before `{`. There are also other places missing spaces. I suggest to use a linter like Eslint.

Reply:
Improved code readability